tfschema/client.go

```diff
@@ -90,8 +90,13 @@ func findPlugin(pluginType string, pluginName string) (*discovery.PluginMeta, er
 func pluginDirs() ([]string, error) {
 	dirs := []string{}
 
+	rootDirectory := os.Getenv("TFSCHEMA_ROOT_DIRECTORY")
```

Comment:
From the point of view of using tfschema as a library, it's hard for the caller to control the behavior if the environment variable is implicitly read in the library code. Some caller may want to call the tfschema multiple times with different directories in a single process.

Add a new `newDefaultClient` method to the `command` package and read it and pass an argument to `NewClient` in the `tfschema` package.
However, we don't want to add more arguments to the `NewClient` method each time we add new features, which would be breaking changes. That's not so fun for the library users.
So we also need add a new `Option` struct to the `tfschema` package and wrap the `rootDir` parameter.

Reply:
I have made the changes in fd6d95a10c6b16eda348ecd90d20c3714bceb932